inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details.html

```diff
@@ -431,7 +431,7 @@
             <button class="btn btn-default"
                     ng-if="vm.record._workflow.status !== 'HALTED'"
                     ng-click="Utils.resumeWorkflow()">
-              <i class="fa fa-play" aria-hidden="true"></i> Resume
+              <i class="fa fa-play" aria-hidden="true"></i> Skip to next step
```

Comment:
When does this happen? If record is not `HALTED` or `COMPLETED` the button will be shown, so if the workflow is ongoing and I press the button what is the effect?

And when is it useful to `Skip to next step` ?